templates/basic-ts/package.json

```
@@ -14,8 +14,9 @@
     "probot-app"
   ],
   "scripts": {
-    "build": "tsc -p tsconfig.json",
-    "dev": "nodemon --exec \"npm start\"",
+    "build": "tsc",
+    "build:watch": "tsc && (tsc -w --preserveWatchOutput & nodemon)",
+    "dev": "nodemon",
```

Upon the application being *create-probot-apped* (initialized), the user should be able to start their application with the `dev` script. You've got the `build:watch` script that does everything `dev` should be doing. Viewing the original source code, it looks like `dev` isn't sufficient for what it's intended to do :fearful:.

This suggestion aliases the `dev` script to the `build:watch` script.

```suggestion
    "dev": "npm run build:watch",
```

Alternate: rename `build:watch` to `dev` and update references to script name.

---

oh okay fair enough 👌